packages/node-provider/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@counterfactual/node-provider",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/src/index.d.ts",
+  "module": "dist/index.es.js",
+  "iife": "dist/index-iife.js",
```

Comment:
Do we need all of these outputs? Specifically `.es` and `.iife`?

Reply:
@snario Not necessarily. I just copied from what we have in `packages/machine`:

https://github.com/counterfactual/monorepo/blob/50b6df871235a9f687a5dd765328c8e4c66a6507/packages/machine/package.json#L1-L7